doc/docs/master/how-tos/use-jupyterhub/index.md

```diff
@@ -1,45 +1,54 @@
-# Use JupyterHub with `python-pachyderm`
+# Use the Pachyderm IDE with `python-pachyderm`
 
 !!! note
-     JupyterHub integration with Pachyderm is an
-     enterprise feature. Contact sales@pachyderm.com
-     to request enabling JupyterHub integration
-     for your Pachyderm Enterprise license.
+    The Pachyderm IDE is an enterprise feature,
+    which is also avaialble for testing during
+    the 14-day free trial.
+    Contact sales@pachyderm.com for more information.
 
 This section describes how you can use the `python-pachyderm`
-client from within the JupyterHub UI.
+client from within the Pachyderm IDE.
 
 !!! note
-    You need to have Pachyderm and JupyterHub installed on the
-    same Kubernetes cluster as described in
-    [Deploy Pachyderm with JupyterHub](../../deploy-manage/deploy/deploy-pachyderm-jupyterhub.md).
+    You need to have Pachyderm IDE installed as described in
+    [Deploy the Pachyderm IDE](../../deploy-manage/deploy/deploy-pachyderm-jupyterhub.md).
 
 ## Overview
 
-JupyterHub is a popular data science platform that enables users
+When you deploy the Pachyderm IDE, you get JupyterHub and
+a customized JupyterLab UI running next to your Pachyderm
+cluster.
+
+Before the proceed, let's clarify the following terms
+as they might be confusing for first-time users:
+
+* JupyterHub is a popular data science platform that enables users
 to quickly spin out multiple single-tenant Jupyter Notebook server instances.
-Jupyter Notebook provides a Python interactive development environment (IDE)
+
+* Jupyter Notebook provides a Python interactive development environment (IDE)
 that is convenient for data science projects. Because of the built-in
 rich-text support, visualizations, the easy-to-use web interface, many
 enterprise users prefer Jupyter Notebooks to the classic Terminal prompt.
 JupyterHub brings all the benefits of Jupyter Notebooks without the need
 to install or configure anything on user machines except for a web browser.
 
-[python-pachyderm](https://github.com/pachyderm/python-pachyderm) is an
+* JupyterLab is a UI that enables you to run multiple Jupyter notebooks and
+other applications in tabs next to each. The tabs can be customized
+to run applications that you need to use as par of your development workflow.
+Pachyderm is one of those applications.
```

Comment:
This isn't quite right, here's my attempt:

JupyterLab is an IDE that allows you to author and run notebooks and code. It is a refresh of the classic Jupyter UI, which some users may be more familiar with.

Reply:
rephrased a bit